commons-lang/src/main/java/org/apache/commons/lang3/mutable/MutableBoolean.java

```diff
@@ -122,10 +122,7 @@ public void setValue(final Boolean value) {
      * @since 2.5
      */
     public boolean isTrue() {
-        if (value)
-            return true;
-        else
-            return false;
+        return value;
```

Comment:
this changes code needs more review

Reply:
@Acesif please do not change code logic just code formatting.